src/PoolTogetherV4.ts

```diff
@@ -0,0 +1,97 @@
+import { providers } from "ethers";
+import { ContractList } from "@pooltogether/contract-list-schema";
+import { Contract } from "@ethersproject/contracts";
+import { createContract, createInterface } from "./utils";
+import { Providers } from "./types";
+import { Provider } from "@ethersproject/abstract-provider";
+import { InfuraProvider } from "@ethersproject/providers";
+import { getAddress } from "@ethersproject/address";
+const debug = require("debug")("v4-js-core");
+export interface PoolTogetherV4Config {
```

Comment:
I think we discussed this in a call a while ago, but provider initialization should happen prior to creation of any classes in this repo. So this isn't needed anymore.